### ISSUE TITLE
"Error Could not get embed link"

### ISSUE THREAD

**Issue by pitsi** [justchokingaround/lobster#161]
**Describe the bug**
As mentioned in the title, the script pops up the message above and exits. It was working until Tuesday 21st, it does not work since Friday 22nd. I do not use it daily though.

**To Reproduce**
1. Launch lobster
2. Search for something
3. Select something from the search results

**Expected behavior**
The script should play whatever was selected.

**Screenshots**
Is a log ok?
[lobster.log](https://github.com/justchokingaround/lobster/files/12706914/lobster.log)

**Desktop (please complete the following information):**
 - OS: Debian testing/unstable x64
 - Player: mpv
 - Version: 4.1.1 (lobster's version)

**Additional context**
The error in the title is what shows up when lobster -x is run, otherwise it just says "error" and exits.

**Comment by medanisjbara** [comment]
Actually the provider itself is down. It is returning 502 (even when you use their crappy frontend)

Hopefully the issue won't persist for long. The dev probably can't do anything about this for now.

**Comment by medanisjbara** [comment]
Actually, setting `provider=Vidcloud` made it work for me. Check the [wiki page](https://github.com/justchokingaround/lobster/wiki/Configuration#provider) if you'd like. I should also note that @justchokingaround is currently working on a PR that brings another provider to the game. But the Vidcloud provider works fine for now.

TL;DR: run `provider=Vidcloud lobster` or `lobster -p Vidcloud` instead of just `lobster` (or set your provider in the config if you'd like, using `lobster -e`)

**Comment by justchokingaround** [comment]
yes, that's the best work around currently ^^^

**Comment by pitsi** [comment]
Thank you for the workaround. Tbh, I should have looked closer when I was troubleshooting it, because e.g. these urls have one vidcloud link that leads nowhere
```
https://flixhq.to/ajax/v2/episode/servers/1373566
https://flixhq.to/ajax/v2/episode/servers/1363765
```
but the one from here works 
`https://flixhq.to/ajax/movie/episodes/19694`
However, its link for upstream is not parsed.

Anyway, that is all for today. We are expecting harsh weather here, for the second time this month. If all goes well, I will be back on Thursday or Friday for more, so please be patient. Thank you :)

p.s. Does the attached log file above return a 3 line fine for anyone else?

**Comment by Workata** [comment]
Could be nice to have a fallback to the next possibly working provider (in case of this issue). Too bad I have no idea how to code in bash 😅

**Comment by pitsi** [comment]
It seems there is a bigger issue today, because an error saying `base64: invalid input` comes up. Here is the relevant log. I have not tried changing the server yet.

[lobster.log](https://github.com/justchokingaround/lobster/files/12726427/lobster.log)

**Comment by pitsi** [comment]
It seems that the problem at the start and the one with base64 no longer happen. If anyone has anything to add, please do.
If not, I will close it tomorrow.

All I want to ask now is which of the 2 providers do you prefer.

**Comment by justchokingaround** [comment]
it seemed to me that `UpCloud| is generally more consistent than `Vidcloud`
